### PR TITLE
added a responsive footer

### PR DESCRIPTION
**Pull request by AbhineshJha**
# Title and Issue number 

Title :Added a responsive footer to the JS challenges

# Checklist:

- [] I have mentioned the issue number in my Pull Request.
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have created a helpful and easy to understand `README.md`
- [] I have updated the Index.html file for my contribution

### CODE REVIEW

**Comment by AbhineshJha** [comment]
hey ! review my pr and add the hacktoberfest-accepted label

File: native/machine-coding/Footer/index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Enhanced Responsive Footer</title>
```
Comment (sadanandpai) [inline native/machine-coding/Footer/index.html:7]: please add all tags needed (refer to other challenges)

File: native/machine-coding/Footer/index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+</head>
+<body>
+    <div class="container">
+        <footer>
+            <div class="footer-content">
+                <div class="footer-logo">
+                    <img src="logo.png" alt="Company Logo">
```
Comment (sadanandpai) [inline native/machine-coding/Footer/index.html:14]: The website main logo can be used here

File: native/machine-coding/Footer/style.css [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    color: #fff;
+}
+
+.footer-bottom {
+    text-align: center;
+    padding-top: 10px;
+}
+
+/* Media query for responsiveness */
+@media screen and (max-width: 768px) {
```
Comment (sadanandpai) [inline native/machine-coding/Footer/style.css:62]: use min-width and lets have mobile-first apporach

**Review by sadanandpai — CHANGES_REQUESTED** [review]

File: native/machine-coding/Footer/index.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Enhanced Responsive Footer</title>
+    <script src="https://kit.fontawesome.com/61673b2d06.js" crossorigin="anonymous"></script>
```
Comment (sadanandpai) [inline native/machine-coding/Footer/index.html:8]: please check the tags from other challenge and include them

**Review by sadanandpai — CHANGES_REQUESTED** [review]

File: native/machine-coding/Footer/index.html (hunk 4 above)
Comment (AbhineshJha) [reply native/machine-coding/Footer/index.html:13]: done

**Review by AbhineshJha — COMMENTED** [review]
Review the Files changed and please add hacktoberfest-accepted label

**Pull request merged**